src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header__images {
+  height: 100%;
+  width: 100%;
+}
+
+.header__nav {
+  height: 100%;
+  /* align-items: center; */
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  text-decoration: none;
+  margin-left: 20px;
+  font-size: 12px;
+  color: #000;
+  text-transform: uppercase;
+  line-height: 60px;
+  text-align: center;
+  align-items: center;
+  white-space: nowrap;
+  cursor: pointer;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
You can create css variable to store this color, as you are using it in several places